src/test/java/uk/gov/hmcts/divorce/testutil/TestConstants.java

```diff
@@ -65,6 +65,8 @@
     public static final String SIGN_IN_DISSOLUTION_TEST_URL = "dissolutionTestUrl";
     public static final String APPLICANT_2_SIGN_IN_DISSOLUTION_TEST_URL = "applicant2DissolutionTestUrl";
 
+    public static final String EMPTY_STRING = "";
```

Comment:
Can remove this now ?

Reply:
Use it here https://github.com/hmcts/nfdiv-case-api/pull/1275/files#diff-f8a9d4709a207b60304bc4fa98c1048e55d6df00e49159036c6dfde3b9213d62R103